website/www/site/layouts/index.html

```diff
@@ -209,16 +209,24 @@ <h2 class="calendar-title">
       {{ T "home-quotes-title" }}
     </div>
 
-    <div class="quotes-desktop swiper">
-      <div class="swiper-wrapper">
-        {{ $data := index $.Site.Data .Site.Language.Lang }}
-        {{ range $quote := $data.quotes }}
-        <div class="swiper-slide">
-          {{ partial "quotes/quote.html" (dict "icon" $quote.icon "text" $quote.text  "logoUrl" $quote.logoUrl "linkUrl" $quote.linkUrl "linkText" $quote.linkText) }}
+    <div class="swiper-container">
+      <div class="swiper-button-prev-custom">
+        <img src="/images/swiper-button-prev.png" alt="previous button">
+      </div>
+      <div class="quotes-desktop swiper">
+        <div class="swiper-wrapper">
+          {{ $data := index $.Site.Data .Site.Language.Lang }}
+          {{ range $quote := $data.quotes }}
+          <div class="swiper-slide">
+            {{ partial "quotes/quote.html" (dict "icon" $quote.icon "text" $quote.text  "logoUrl" $quote.logoUrl "linkUrl" $quote.linkUrl "linkText" $quote.linkText) }}
+          </div>
+          {{ end }}
         </div>
-        {{ end }}
+        <div id="swiper-pagination-id" class="swiper-pagination"></div>
```

Comment:
can this div be shortened?

`<div id="swiper-pagination-id" class="swiper-pagination" / >`

Reply:
the layout somehow breaks, when i try to use the shortened variant <div id="swiper-pagination-id" class="swiper-pagination" />